.github/labeler.yml

```
@@ -20,6 +20,10 @@
 - changed-files:
   - any-glob-to-any-file: "csi/**"
 
+"Area/Jobs/async-upload":
```

```suggestion
"Area/Jobs/Async-upload":
```

wdyt? the other labels seems sorta of pascal case

---

yeah we also have some all UPPERCASE but I'm fine either way :)